src/basic/strv.h

```diff
@@ -69,8 +69,10 @@ bool strv_equal(char **a, char **b);
 char **strv_new(const char *x, ...) _sentinel_;
 char **strv_new_ap(const char *x, va_list ap);
 
+#define STRV_NULL ((const char *) -1)
```

Comment:
i wonder if STRV_NULL is really the right name for this. After all, it's not NULL at all, but quite the opposite, with -1. Maybe call it STRV_NIL or STRV_SKIP or so? STRV_IGNOREME might work too?